lib/eventApi.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (app) {
 
     app.get('/events/:name', function (req, res) {
         eventDb.getEventsFilterByName(req.params.name).then(function (events) {
-            if (events) {
+            if (events.length > 0) {
```

Comment:
Events was always true. This was just a test to prevent it to return empty results. Will change anyway in the rewrite of the API